docs/Community/Governance.md

```diff
@@ -0,0 +1,137 @@
+Cross-SIG efforts can be coordinated by establishing a Working Group (WG), which
+is given a charter and which reports progress back to the relevant SIG leaders
+or the CGC.
+
+If there is conflict, either within or between SIGs, or between any members of
+the community, wherein it is desirable, the issue can be raised to the CGC.
+
+Technical Discussions
+---------------------
+
+Technical discussions generally happen in one of these places:
+
+* [on the GitHub project](https://github.com/openenclave/openenclave), either in
+  an Issue or a Pull Request
+* in the public meetings of Committees, Special Interest Groups, or Working Groups, which can be found via the [community calendar](https://lists.confidentialcomputing.io/calendar).
+* on the community mailing list or chat server, which is detailed in the [Comminucations](Communication.md) document
+
+While we strive for openness and transparency, it is natural that some
+discussions will also happen in private channels, or in impromptu face-to-face
+conversations (for those who work in the same building, or bump into each other
+at a conference). The outcome of such discussions should be considered
+non-binding until the information is shared through an open medium and relevant
+stake-holders have opportunity to contribute to the discussion.
+
+All SIG and WG leaders are responsible for creating and publishing minutes from
+the meetings they hold, ensuring that the meetings are open and inclusive, and
+that decisions they make are fair and transparent.
+
+If a private discussion pertains to an open issue or PR, and results in a
+decision, that item should be updated and reference given to the conversation.
+
+Issue/PR Review Process
+-----------------------
+
+The community strives to triage and respond to all incoming issues within one
+week. If you haven't heard anything by then, or an issue or PR is not getting
+any response within a reasonable time after it is triaged, please feel free to
+remind us with a ping on the thread.
+
+Security issues should be reported through a separate channel, and
+will receive a response within 24 hours. See [Reporting Security
+Issues](Contributing.md#reporting-security-issues).
+
+Committers and Contributors
+---------------------------
+
+At present, we maintain a list of [Project Committers](Committers.md) along with
+descriptions of their knowledge areas.
+
+This will be updated and replaced with a more scaleable approach to
+commit-rights management.
+
+Accepting Contributions
+-----------------------
+
+SIGs will generally accept changes that improve the project and that align with the
+[Open Enclave roadmap](https://github.com/openenclave/openenclave/projects).
+
+All contributions are expected to be reviewed by at least one SIG lead other
+than the contributor, and have no significant objections from other SIG leads
+within the same SIG.
+
+If a change requires the approval of multiple SIGs (e.g., because it affects
```

Comment:
Tangent: Does this imply the adoption of new tooling that supports this semantic? (e.g. prow & LGTM)

Reply:
If bors does not support this requirement, then yes

As I understand it, Prow was build out of the needs of another distributed, multi-party, github-driven workflow, very similar to (if not exactly the same as) what I believe OpenEnclave will need as we grow.